src/SFA.DAS.FAA.Web/Models/Applications/IndexViewModel.cs

```diff
@@ -0,0 +1,95 @@
+﻿using SFA.DAS.FAA.Application.Queries.Applications.GetIndex;
+using System.Globalization;
+using SFA.DAS.FAA.Web.Extensions;
+using SFA.DAS.FAT.Domain.Interfaces;
+
+namespace SFA.DAS.FAA.Web.Models.Applications
+{
+    public class IndexViewModel
+    {
+        public ApplicationsTab SelectedTab { get; set; }
+        public string PageTitle { get; set; }
+        public string TabTitle { get; set; }
+        public string TabText { get; set; }
+        public bool IsTabTextInset { get; set; }
+        public string TabTextStyle => IsTabTextInset ? "govuk-inset-text" : "govuk-body";
+
+        public List<Application> Applications { get; set; } = [];
+        public List<Application> ExpiredApplications { get; set; } = [];
+
+        public class Application
+        {
+            public Guid Id { get; set; }
+            public string VacancyReference { get; set; }
+            public string Title { get; set; }
+            public string EmployerName { get; set; }
+            public string StartedOn { get; set; }
+            public string ClosingDate { get; set; }
+            public bool IsClosingSoon { get; set; }
+            public bool IsClosed { get; set; }
+        }
+
+        public static IndexViewModel Map(ApplicationsTab tab, GetIndexQueryResult source, IDateTimeService dateTimeService)
+        {
+            var result = new IndexViewModel
+            {
+                SelectedTab = tab,
+                PageTitle = $"{tab.GetTabTitle()} applications"
+            };
+
+            foreach (var application in source.Applications.OrderByDescending(x => x.CreatedDate))
+            {
+                var timeUntilClosing = application.ClosingDate.Date - dateTimeService.GetDateTime();
+                var daysToExpiry = (int)Math.Ceiling(timeUntilClosing.TotalDays);
+
+                var closingDate = "";
+                switch (daysToExpiry)
+                {
+                    case < 0:
+                        closingDate = $"Closed on {application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)}";
+                        break;
+                    case 0:
+                        closingDate = "Closes today at 11:59pm";
+                        break;
+                    case 1:
+                        closingDate = $"Closes tomorrow ({application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm)";
+                        break;
+                    case <= 31:
+                        closingDate = $"Closes in {daysToExpiry} days ({application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm)";
+                        break;
+                    default:
+                        closingDate = $"Closes on {application.ClosingDate.ToString("dddd d MMMM yyyy", CultureInfo.InvariantCulture)} at 11:59pm";
+                        break;
+                }
```

Comment:
i feel this may need to come out at some point, I think we have other things doing similar around text and closing dates 🤔